v63004/inref/gtm8643.m

```diff
@@ -11,33 +11,18 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;
 gtm8643
- 	set tcpdev="server$"_$j
- 	set timeout=10
  	set tcpdev="server$"_$j
  	set portno=0
+	set timeout=10
```

Comment:
How long does the test take to run? Does it run faster with a timeout of 1 (instead of 10 seconds)? If so, I would prefer 1. If not, then I am fine with the current changes.

Reply:
So it seems like the timeout variable is never used to wait for that amount of time, but is perhaps used to set the tcpdev up. The test ran in 13.66 seconds with the timeout=10 and 13.95 seconds with timeout=1. So it does not seem to have any effect on the run time.

Reply:
Then I am fine with the current changes.